### PR TITLE
Add informations to publish the ExCorreios to Hex.pm

### PR DESCRIPTION
**Pull request by Bscruz19**
## Objective

Add necessary information to publish the lib to Hex.pm

### Reviewers

@locaweb

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -1,14 +1,25 @@
...
-      version: "1.1.3",
+      deps: deps(),
       elixir: "~> 1.10",
       elixirc_paths: elixirc_paths(Mix.env()),
       start_permanent: Mix.env() == :prod,
-      deps: deps()
+      version: @version,
+
+      # Hex
+      description: "",
```
Comment (rhruiz) [inline mix.exs]: Maybe we could use the README description as the base for this? Something like

Elixir client that integrates with Brazilian Correios API.

File: README.md [hunk 2]
```diff
@@ -294,3 +294,19 @@ Credo is a static code analysis tool for the Elixir language, to run credo:
...
+
+## Contributing
+
+Check out [Contributing](CONTRIBUTING.md) guide.
+
+## License
+
+MIT license
+
+Copyright (c) 2015-2016 Locaweb
```
Comment (rhruiz) [inline README.md]: 2020 here :P

Comment (Bscruz19) [reply README.md]: Fixed XD

File: mix.exs (hunk 1 above)
Comment (Bscruz19) [reply mix.exs]: Great!!! :) 
Fixed!!!

**Pull request merged**